artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/jms/ServerJMSMapMessage.java

```diff
@@ -137,84 +123,75 @@ public void setObject(final String name, final Object value) throws JMSException
          }
          TypedProperties.setObjectProperty(new SimpleString(name), val, map);
       } catch (ActiveMQPropertyConversionException e) {
-         throw new MessageFormatException(e.getMessage());
+         throw new RuntimeException(e.getMessage());
       }
    }
 
-   @Override
-   public boolean getBoolean(final String name) throws JMSException {
+   public boolean getBoolean(final String name) throws Exception {
       try {
          return map.getBooleanProperty(new SimpleString(name));
       } catch (ActiveMQPropertyConversionException e) {
-         throw new MessageFormatException(e.getMessage());
+         throw new RuntimeException(e.getMessage());
       }
    }
 
-   @Override
-   public byte getByte(final String name) throws JMSException {
+   public byte getByte(final String name) throws Exception {
       try {
          return map.getByteProperty(new SimpleString(name));
       } catch (ActiveMQPropertyConversionException e) {
-         throw new MessageFormatException(e.getMessage());
+         throw new RuntimeException(e.getMessage());
```

Comment:
-1 just throwing a runtime exception. This loses all context about the exception type. Ditto this on all same in this pr

Reply:
This should not really happen at the Converters.

Creating our own Exception here wouldn't make it more useful.

Reply:
Should not happen doesnt mean it cannot...

Reply:
It can... a RuntimeException that's captured on the converter is all we need.

What good would make if I created a generic ConversionException and replaced all the JMSException by ConversionException here?

Reply:
You may as well just let the original exception bubble out.

Reply:
Least that way youd get something a little more useful and specific...in this sample "ActiveMQPropertyConversionException"